atest/testdata/standard_libraries/builtin/repeat_keyword.robot

```diff
@@ -89,3 +96,8 @@ First Continuable Failure And Then Pass Execution
     Pass Execution If    "${PASS EXECUTION}" == "yes"    Message
     Set Suite Variable    ${PASS EXECUTION}    yes
     Run Keyword And Continue On Failure    Fail    Continuable
+
+Keyword Executing For 1 second
+    [Arguments]     ${TIME FORMAT}
+    Log    This is done for ${TIME FORMAT}
+    Sleep    1 seconds
```

Comment:
To check that "Repeat Keyword" actually repeats the keyword, I needed to be sure that the keyword runs several times. Because just "several times" cannot be the criterion for the test, I wanted to repeat keyword as number of times as I intended.
If the keyword sleep for 1 second at once, and "Repeat Keyword" repeats it for 3 seconds, the keyword should run 3 times approximately.

I will modify that the keyword sleeping 0.1 second and repeating it for 0.3 second.
Is it OK? Any other suggestions?